servers/auth/README.md

```diff
@@ -0,0 +1,71 @@
+# Traefik Auth Microservice - Demo
+
+A basic demo version of the Traefik proxy & Authentication microservice is presented here. 
+This demo uses docker for running the microservice. Within docker, images of the following container are used:
+
+- traefik - As the reverse proxy/ edge of our system
+- thomseddon/traefik-forward-auth - That works as an authentication server, with Gitlab as the OAuth provider. 
+- whoami - A tiny server which return information about your OS/IP. This is an example version of any microservice that should lie behind the OAuth.
+
+## Example Rules
+
+- The objective is to be able to acces the information returned by the whoami server. This information is available as http://localhost/os , but only after successful OAuth. 
+
+- The OAuth works on the simple rule that any "xyz@gmail.com" email IDs registed with Gitlab are allowed. Any other gitlab accounts, like "pqr@domain.com" (where domain is not gmail) are not allowed. 
```

Comment:
Please replace gmail.com with localhost to reflect the existing rules in `conf`. You can move the existing explanation to next paragraph and mention the required changes to be made in the conf file. You can provide an example conf file within README itself.